src/types/cookie.ts

```diff
@@ -0,0 +1,7 @@
+/** The representation of a note on the client. */
+export interface CookieClientModel {
```

Comment:
You can just name it `CookieConsent` or something like that. `Client` and `Server` in the name were just used to easier differentiate the models of the server and the client. The cookie consent is something that will only be stored on the client so the name is not necessary. Also the term `consent`, which is defined by the `value` is missing in the name.

Reply:
... i guess this interface will be deleted anyway if you delete the redux stuff. So the comment above is just jFYI :)